ngraph_bridge/enable_variable_ops/ngraph_rewrite_for_tracking.cc

```diff
@@ -27,6 +28,32 @@ namespace tensorflow {
 
 namespace ngraph_bridge {
 
+// returns true if all the output nodes of this node
+// are implemented on bridge
+bool AreOutputsNGSupported(Node* node) {
+  for (auto edge : node->out_edges()) {
+    if (edge->dst()->IsOp() && !edge->IsControlEdge() &&
+        !IsNGSupportedType(edge->dst()->type_string())) {
+      NGRAPH_VLOG(5) << "ngraph does not support dst node "
+                     << edge->DebugString();
+      return false;
+    }
+  }
+  return true;
+}
+
+// returns true if this node is a static input to any of its output nodes
+bool NodeIsStaticInput(Node* node) {
+  for (auto edge : node->out_edges()) {
+    if (edge->dst()->IsOp() && !edge->IsControlEdge() &&
+        InputIsStatic(edge->dst(), edge->dst_input())) {
+      NGRAPH_VLOG(5) << "ngraph does not support dst node ";
```

Comment:
dst node is not supported because it has a static input

Reply:
stray comment. will fix it